src/Controllers/HistoryViewerController.php

```diff
@@ -32,12 +35,23 @@ class HistoryViewerController extends LeftAndMain
         'schema',
     ];
 
+    /**
+     * An array of supported form names that can be requested through the schema
+     *
+     * @var string[]
+     */
+    protected $formNames = ['versionForm', 'compareForm'];
+
     public function getClientConfig()
     {
         $clientConfig = parent::getClientConfig();
 
         $clientConfig['form']['versionForm'] = [
-            'schemaUrl' => $this->owner->Link('schema/versionForm')
+            'schemaUrl' => $this->owner->Link('schema/versionForm'),
+        ];
+
+        $clientConfig['form']['compareForm'] = [
+            'schemaUrl' => $this->owner->Link('schema/compareForm'),
```

Comment:
Rather than rebase, it would probably be easier to merge the base branch and not rewrite history. We can rebase the whole thing onto silverstripe/versioned-admin later.